specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -1721,10 +1783,14 @@
           ],
           "type": "string",
           "x-ms-enum": {
-            "name": "managedResourcesPublicNetworkAccess",
+            "name": "PublicNetworkAccess",
             "modelAsString": true
           }
         },
+        "mergeInfo": {
+          "description": "Gets or sets the Merge Info.",
```

Comment:
IMO the doc could probably be more verbose to provide more context

Reply:
I will add this into the above work item, but would like to avoid pushing additional changes at this time if not required.